src/main/kotlin/uk/gov/justice/digital/hmpps/incentivesapi/service/PrisonerIepLevelReviewService.kt

```diff
@@ -322,6 +322,23 @@ class PrisonerIepLevelReviewService(
       prisonerNumber = prisonerNumber,
       auditModuleName = "Incentives-API",
     )
+  @Transactional
+  suspend fun processMergedPrisoner(prisonerMergeEvent: PrisonerMergeEvent) {
+    val numberUpdated = prisonerIepLevelRepository.updatePrisonerNumber(
+      prisonerMergeEvent.additionalInformation.removedNomsNumber,
+      prisonerMergeEvent.additionalInformation.nomsNumber
```

Comment:
I think these are the wrong way around? I think they should be as per below - I've added in the method param names to the call arguments which may help
```
val numberUpdated = prisonerIepLevelRepository.updatePrisonerNumber(
  remainingPrisonerNumber = prisonerMergeEvent.additionalInformation.nomsNumber,
  removedPrisonerNumber = prisonerMergeEvent.additionalInformation.removedNomsNumber
)
```